evalml/tests/utils_tests/test_gen_utils.py

```diff
@@ -215,17 +215,17 @@ def _check_equality(data, expected, check_index_type=True):
         pd.testing.assert_frame_equal(data, expected, check_index_type)
 
 
-@pytest.mark.parametrize("data,num_to_pad,expected",
-                         [(pd.Series([1, 2, 3]), 1, pd.Series([np.nan, 1, 2, 3])),
+@pytest.mark.parametrize("data, num_to_pad, expected",
+                         [(pd.Series([1, 2, 3]), 1, pd.Series([np.nan, 1, 2, 3], dtype="Float64")),
```

Comment:
You can also do `dtype=pd.Float64Dtype()` to make it clear that you are using the new Nullable Float Dtype/.

Note that would be different than `dtype=float64` (regular, pandas float64, not new dtype)

Reply:
Do you know if it's any different from using the string or is it more stylistic in nature? I was more inclined to use the string version since that's primarily what we've been doing for "Int64" but could file something to update our instances to use an instance of the actual pandas dtype object :)

Reply:
Not sure if it's any different, it might be more stylistic in nature. You can also do `pd.Int64Dtype()`